src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/vmware/VMWareProviderVirtualMachineUtil.java

```diff
@@ -9,24 +9,32 @@
 import org.springframework.stereotype.Component;
 
 import com.vmware.vim25.InvalidProperty;
-import com.vmware.vim25.ManagedObjectReference;
 import com.vmware.vim25.RuntimeFault;
 import com.vmware.vim25.VirtualMachineRelocateSpec;
 import com.vmware.vim25.mo.Folder;
 import com.vmware.vim25.mo.InventoryNavigator;
 import com.vmware.vim25.mo.ManagedEntity;
-import com.vmware.vim25.mo.ResourcePool;
 import com.vmware.vim25.mo.VirtualMachine;
 
 
 @Component
-public class VMWareProviderVirualMachineUtil {
+public class VMWareProviderVirtualMachineUtil {
```

Comment:
Static classes make things very hard to mock and test. I think in this case is ok to leave it like it is now.